prov/psm/src/psmx_init.c

```diff
@@ -558,6 +558,7 @@ static int psmx_getinfo(uint32_t version, const char *node, const char *service,
 	psmx_info->domain_attr->mr_iov_limit = 65535;
 	psmx_info->domain_attr->caps = PSMX_DOM_CAPS;
 	psmx_info->domain_attr->mode = 0;
+	psmx_info->domain_attr->mr_cnt = 10000;
```

Comment:
That's just the maximum value of the mr key, and MRs can't share the same key .... in practical it's unlimited, the value is not treated differently.